openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -446,6 +449,11 @@ The request parameter `authorization_details` defined in Section 2 of [@!RFC9396
 * `credential_configuration_id`: REQUIRED when `format` parameter is not present. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format-profiles). It MUST NOT be present if `format` parameter is present.
 * `format`: REQUIRED when `credential_configuration_id` parameter is not present. String identifying the format of the Credential the Wallet needs. This Credential Format Identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format-profiles). It MUST NOT be present if `credential_configuration_id` parameter is present.
 
+Additional `authorization_details` parameters MAY be defined and used,
+as described in [@!RFC9396],
+including when the `type` value is `openid_credential`.
```

Comment:
```suggestion
Additional `authorization_details` parameters MAY be defined and used,
when the `type` value is `openid_credential`, as described in [@!RFC9396].
```
I don't think we want to allow new `type` values? even if we do, this text should be clearer if we allow new type values or more extensions to type openid_credential

Reply:
The spec says this, so it's already clear what the type value must be when using it:
* `type`: REQUIRED. String that determines the authorization details type. It MUST be set to `openid_credential` for the purpose of this specification.

Therefore, I believe it was unnecessary for me to say anything about the type value.  I propose to simply delete the qualifier, so the sentence is more parallel to other such sentences in the spec, and simply becomes:
> Additional `authorization_details` parameters MAY be defined and used, as described in [@!RFC9396].